README.md

```diff
@@ -2,23 +2,163 @@
+3. Follow usage instructions above using the newly created `build/iplboot.gcb`.
+
+### Qoob SX
+
+NOTE: Qoob SX is currently nonfunctional as the resulting BIOS is too large.
+
+1. Ensure dolxz is installed (see [instructions](#installing-dolxz) below).
+2. Ensure doltool is installed (see [instructions](#installing-doltool) below).
+3. Obtain `qoob_sx_13c_upgrade.elf` and copy it to the root of this project. (TODO: MD5 and more details about the file.)
+4. Run `make qoobsx`.
+5. Follow usage instructions above using the newly created `build/qoob_sx_iplboot_upgrade.elf`
+
+### Viper
+
+1. Run `make viper`.
+2. Follow usage instructions above using the newly created `build/iplboot.vgc`.
+
+### GameCube Memory Card
+
+1. Ensure dol2gci is installed (see [instructions](#installing-dol2gci) below).
+2. Run `make gci`.
+3. Follow usage instructions above using the newly created `build/iplboot.gci`.
+
+*Optional:* If you want to reduce the file size by ~50% so it will take up less space on your memory card, you can create a compressed version.
+
+1. Ensure dol2gci is installed (see [instructions](#installing-dol2gci) below).
+2. Ensure dolxz is installed (see [instructions](#installing-dolxz) below).
+3. Run `make gci_compressed`.
+4. Follow usage instructions above using the newly created `build/iplboot_xz.gci` (Notice the `_xz` suffix).
+
+### Compressed DOL
+
+Should you need it, you can create a compressed version of the base DOL.
+
+1. Ensure dolxz is installed (see [instructions](#installing-dolxz) below).
+2. Run `make dol_compressed`.
+3. Follow usage instructions above using the newly created `build/iplboot_xz.dol` (Notice the `_xz` suffix).
+
+
+## Dependencies
+
+**NOTE:** Some of these are required only in certain scenarios and you may not need them. Check the instructions for your target device first.
+
+### Installing libogc2
+
+1. Ensure you have the latest version of [devkitPro](https://devkitpro.org/wiki/Getting_Started) installed.
+2. Clone or download the [libogc2 repo](https://github.com/extremscorner/libogc2).
+3. Run `make install`.
+
+### Installing dolxz
+
+1. Ensure you have the latest version of [devkitPro](https://devkitpro.org/wiki/Getting_Started) installed.
+2. Clone or download the [dolxz repo](https://github.com/yo1dog/dolxz).
+3. Run `make`.
+4. Ensure the resulting `dolxz` is executable (`chmod +x doltool`).
+5. Ensure `dolxz` is accessible from your `PATH` env var.
 
-After flashing, place an ipl.dol file on your SD card and turn the Cube on, it
-will load it right away. The IPL also acts as a server for emu_kidid's
-[usb-load](https://github.com/emukidid/gc-usb-load), should you want to use it
-for development purposes.
+### Installing dol2gci
 
-## Building
+1. Download from [here](https://github.com/emukidid/swiss-gc/blob/master/buildtools/dol2gci?raw=true). Alternatively, you can clone or download the [swiss repo](https://github.com/emukidid/swiss-gc/tree/master/buildtools) and build yourself.
```

Comment:
Don't link directly to Linux binaries. There are Windows versions right alongside them. I haven't personally tested building on Windows but I don't want to make it any harder than necessary.